packages/cerebral/src/Controller.js

```diff
@@ -70,7 +70,10 @@ class Controller extends EventEmitter {
       console.warn('You are not using the Cerebral devtools. It is highly recommended to use it in combination with the debugger: https://cerebral.github.io/cerebral-website/install/02_debugger.html')
     }
 
-    if (this.options.strictRender) {
+    if (
+      process.env.NODE_ENV !== 'production' &&
```

Comment:
there is isDeveloping utility in util.js
https://github.com/cerebral/cerebral/blob/master/packages/cerebral/src/utils.js#L72